### DISCUSSION TITLE
Room in ursina

### DISCUSSION THREAD

**Discussion by ghost** [pokepetter/ursina#131] [Q&A]
Hi, Iam new with urisna. Its great tool but i wane ask you have to make room?
Because, i know, i can do it with four cubes but its not so good way to do that because now i have 36 sizes.
So is there way to make something like empty cube, and than put FirstPersonController in to that cube?

**Comment by IndividualCoder** [comment]
just make a big cube and give it the property "double_sided = True" and it will render from inside.
